### DISCUSSION TITLE
Wireguard docs is big "huh???"

### DISCUSSION THREAD

**Discussion by scottw-finao** [netdata/netdata#19170] [Q&A]
I was looking for a tool to monitor wireguard connections. I installed netdata, verified it worked, went to the documentation. It tells me to go into the config directory, run some edit script pointing at the wireguard configuration file. It opens up and it's empty.

The documentation shows absolutely no examples of what is supposed to be in this file.  How is one supposed to now how to configure wireguard with zero content and zero examples in the documentation?

**Comment by ilyam8** [comment]
There are [no configuration examples](https://learn.netdata.cloud/docs/collecting-metrics/vpns/wireguard#examples) because the WireGuard collector doesn't have any configuration parameters other than `update_every` (data collection interval).

**Comment by scottw-finao** [comment]
um ok - but I open netdata in a browser and see nothing wireguard oriented in it. I set wireguard=yes in the main parameters and restarted.

**Reply by ilyam8** [reply]
Could you let me know where you checked? It should be under System->Network->VPN.

And don't forget about Wireguard's collector [troubleshooting documentation](https://learn.netdata.cloud/docs/collecting-metrics/vpns/wireguard#troubleshooting).

**Comment by scottw-finao** [comment]
I abandoned even trying to use this tool since there was no reference information available.  I don't have time to go hunting for 'how to use' a supposed-to-be-plug-n-play tool.
